content/en/docs/guides/complex-component-structure-deployment.md

```diff
@@ -0,0 +1,839 @@
+---
+title: "Deployment scenario using a complex component"
+description: "Deploying a complex microservice architecture component."
+date: 2023-06-20T10:00:00+00:00
+lastmod: 2023-06-20:00:00+00:00
+draft: true
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "gitops-driven-configuration-of-ocm-applications"
+weight: 115
+toc: true
+---
+
+## Prerequisite
+
+We assume that the reader has already read all the previous guides in the component area. This guide discusses a more
+complex scenario using plain Localizations and Configurations without the use of Unpacker.
+
+## Constructing the Component
+
+We are going to use [podinfo](https://github.com/stefanprodan/podinfo) in microservices mode. This enables us to deploy several components and configure them
+individually.
+
+Podinfo has three components which we are going to put into individual component descriptors.
+- backend
+- frontend
+- cache (redis)
+
+We will use the following example application to demonstrate multi-component structure: [Podinfo Component](https://github.com/open-component-model/podinfo).
+
+This repository contains the following items:
+
+### Makefile
+
+A `Makefile` that can be used to build the component and push it to a user repository.
```

Comment:
if we are going to show how to build the component, i think a component file would be the better approach.

Reply:
I also think providing the needed file structure and `ocm cli` commands in order to build the component is better for demonstration purpose. The `makefile` is another level of complexity for beginners.

Reply:
Fair point. However, at this point, there were several guides introducing the OCM CLI functionality. Do we really want to repeat that here? Shouldn't the reader be already aware of them?

Reply:
I second @souleb here and would also be more verbose using ocm tools native command. Let's better repeat something that we already mentioned at another place, than to introduce another level of complexity with the makefile.

Reply:
Okay then.